lightning_integration_test.py

```diff
@@ -3,14 +3,19 @@
 # NOTE: subprocess.run requires Python 3.5+
 import subprocess
```

Comment:
Does just having `from subprocess import run` work, or do we need both `import subprocess` and `from subprocess import run`?